ak_survey_results.py

```diff
@@ -51,13 +47,14 @@ def __init__(self, settings):
         """
         Initialize settings.
         """
+        db_settings = get_secret('redshift-admin')
         self.settings = settings
         self.database = psycopg2.connect(
-            host=self.settings.DB_HOST,
-            port=self.settings.DB_PORT,
-            user=self.settings.DB_USER,
-            password=self.settings.DB_PASS,
-            database=self.settings.DB_NAME
+            host=db_settings['host'],
+            port=db_settings['port'],
+            user=db_settings['username'],
+            password=db_settings['password'],
+            database='dev'
```

Comment:
After you left the meeting, we ended up adding database name as a secret value, under the name `dbName`. Even though it's not a real secret, we thought it made more sense to keep it with the other database connection parameters to be fetched where needed, rather than requiring it to be passed in via a separate configuration file for scripts or functions that might support multiple different databases.

It's fine to keep it hard coded here, I just thought I'd mention it in case you wanted to use `db_settings['dbName']` here instead.